metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFastPartitionService.java

```diff
@@ -75,7 +76,7 @@
  * @since 1.0.0
  */
 @Slf4j
-@Transactional(readOnly = true)
+@Transactional("hiveTxManager")
```

Comment:
If all the methods in this class are readOnly why not leave the readOnly at the class level and remove the redundancy at the method level? Are you just concerned if for some reason a method is ever introduced that wasn't read only and so you're opting methods into readOnly mode instead of other way around?

Reply:
Yes. Explicitly set readOnly at the method level if the method only does a read.